### PR TITLE
feat(odyssey-react): add hide/show for password input

### PR DESCRIPTION
**Pull request by sunnylittle-okta**
### Description
Draft PR, but publishing for deployed storybook, includes changes from dependent PR https://github.com/okta/odyssey/pull/1406
Added a suffix button when the input type is "password" for hiding and showing of the password

### CODE REVIEW

**Comment by josesolano-okta** [comment]
Looks really good. I think this one is a bit different from clearing search because when clicking the show/hide button I think you should remain focused on the button and not go back to the input.
@edburyenegren-okta what do you think?

File: packages/odyssey-react/src/components/TextInput/TextInput.tsx [hunk 1]
```diff
@@ -249,13 +302,14 @@ export const TextInput = withTheme(
...
         <div className={rootStyles}>
+          {/** Text or Icon Prefix */}
           {(prefix || isPrefixIcon) && (
             <span
               className={prefixStyles}
               aria-hidden="true"
               onClick={setFocus}
             >
-              {isPrefixIcon ? <Icon name={prefixIconName} /> : prefix}
+              {isPrefixIcon ? <Icon name={prefixIcon} /> : prefix}
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/TextInput/TextInput.tsx:312]: see my feedback in the other PR about the build outcome for using `Icon` I think we may want to reach for a new type annotation here for prefix and suffix that allow users to enter their own icon via composition

**Review by jeffbelser-okta — COMMENTED** [review]
strong work 💪  one comment inline and watch out for `fix()...` commits that are only affecting code in your feature branch those can be `chore` if so or squashed

File: packages/odyssey-react/src/components/Icon/EyeOff.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            fill="currentColor"
+          />
+        </svg>
+      </SvgIcon>
+    );
+  }
+);
+
```
[the remark this answers is not in the document]
Comment (sunnylittle-okta) [reply packages/odyssey-react/src/components/Icon/EyeOff.tsx:39]: no, i couldn't find anything in the readme's. I just looked at the package.json's.
Hope I did it correctly. 😬

File: packages/odyssey-react/src/components/TextInput/TextInput.tsx (hunk 1 above)
Comment (sunnylittle-okta) [reply packages/odyssey-react/src/components/TextInput/TextInput.tsx:312]: Noted, I like the usage of `Icon` with name but I get it....

File: packages/odyssey-react/src/components/Tabs/Tabs.module.scss [hunk 3]
```diff
@@ -29,11 +29,30 @@
   color: var(--TabTextColor);
   font-family: inherit;
   font-size: var(--TabFontSize);
-  font-weight: var(--TabFontWeight);
```
Comment (edburyenegren-okta) [inline packages/odyssey-react/src/components/Tabs/Tabs.module.scss:32]: Wassup with these Tab changes?

Comment (sunnylittle-okta) [reply packages/odyssey-react/src/components/Tabs/Tabs.module.scss:32]: I think you won't see this in my PR once I point to `develop`

Comment (edburyenegren-okta) [reply packages/odyssey-react/src/components/Tabs/Tabs.module.scss:32]: Ah, just realized this was a feature-to-feature PR! 😅

**Pull request closed without merging**